Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js

```diff
@@ -290,8 +289,8 @@ function polydataUpdater(instance, state, context) {
       .then(
         (values) => {
           const array = vtkDataArray.newInstance(Object.assign({ values }, arrayMetadata));
-          // console.log('array', array.getName());
-          arraysToBind.push([instance.get(arrayMetadata.location)[arrayMetadata.location].addArray, [array]]);
+          const regMethod = arrayMetadata.registrationMethod ? arrayMetadata.registrationMethod : 'addArray';
```

Comment:
I don't know if we want the same name as [here](https://github.com/Kitware/vtk-js/blob/master/Sources/IO/Core/HttpDataSetReader/index.js#L62)

I mean, I like the explicit part of `registrationMethod` vs `registration`, but just wondering.